test/smoke/framework/operator.go

```diff
@@ -42,10 +42,31 @@ const (
 	kogitoInfraCrdName       = "kogitoinfras"
 	kogitoDataIndexCrdName   = "kogitodataindices"
 	kogitoJobsServiceCrdName = "kogitojobsservices"
+
+	kogitoOperatorTimeoutInMin = 5
+
+	communityCatalog = "community-operators"
 )
 
+type dependentOperator struct {
+	timeoutInMin int
+	channel      string
+}
+
 var (
 	defaultOperatorImageTag = version.Version
+
+	//kogitoOperatorCommunityDependencies = []string{"infinispan", "strimzi"}
+	kogitoOperatorCommunityDependencies = map[string]dependentOperator{
```

Comment:
I guess you should add Keycloak here as well? Since it's on the CSV file.

Reply:
Wanted to set Keycloak tests with this issue: https://issues.redhat.com/browse/KOGITO-912